### PR TITLE
Added remove-configuration and updated README.md to reflect it

### PR DESCRIPTION
**Pull request by Dantheman720**
Please let me know if you need me to make any changes.

Thanks!
Dan

### CODE REVIEW

**Comment by sgilda** [comment]
Nice job!! A few minor adjustments and this is good to go!

File: ejb-security-propagation/README.md [hunk 1]
```diff
@@ -416,6 +416,27 @@ You can remove the security domain configuration by manually restoring the back-
...
+### Remove the Security Domain Configuration by Running the JBoss CLI Script
+
+1. Start the JBoss Enterprise Application Platform 6 Server by typing the following: 
+
+        For Linux:   JBOSS_HOME/bin/domain.sh
+        For Windows: JBOSS_HOME\bin\domain.bat
+2. Open a new command line, navigate to the root directory of this quickstart, and run the following command, replacing JBOSS_HOME with the path to your server:
+
+        JBOSS_HOME/bin/jboss-cli.sh --connect --file=remove-configuration.cli 
+This script removes the log and file handlers from the `security domain` subsystem in the server configuration. You should see the following result when you run the script:
```
Comment (rafabene) [inline ejb-security-propagation/README.md]: This description differs from https://github.com/jboss-jdf/jboss-as-quickstart/pull/606/files#L1R1 - "script to remove the quickstart-domain security domain from the JBoss server". It doesn't seem to remove any log and file handlers

Comment (sgilda) [inline ejb-security-propagation/README.md]: I think you can make it easy on yourself and say:

This script removes the server configuration that was done by the `configure-server.cli` script. You should see the following result when you run the script:

Comment (sgilda) [inline ejb-security-propagation/README.md]: Or:

This script restores the server configuration to its state before running the `configure-server.cli` script.

Comment (Dantheman720) [inline ejb-security-propagation/README.md]: That works for me.  I will make the adjustment.

**Comment by rafabene** [comment]
Now it looks good. Do you want me to merge ?

**Comment by sgilda** [comment]
Sorry. I already did it, but forgot to close this!

**Pull request closed without merging**